radixdlt/src/integration/java/com/radixdlt/network/transport/udp/NettyUDPTransportTest.java

```diff
@@ -41,6 +42,8 @@
 import static org.junit.Assert.*;
 
 public class NettyUDPTransportTest {
+	private static final int TEST_SECONDS = SimulationTest.getConfiguredDuration()
+			.map((value, unit) -> (int)((unit.toSeconds(value) + 1)/2));
```

Comment:
Not a big deal, but we usually use single tab continuation indent.

Reply:
Fixed.